Assets/MixedRealityToolkit.Examples/Demos/StandardShader/Scripts/MaterialParameterChanger.cs

```diff
@@ -0,0 +1,532 @@
+    /// </summary>
+    public class MaterialParameterChanger : MonoBehaviour
+    {
+        /// <summary>
+        /// parameter
+        /// </summary>
+        //TargetMateria;
+        public Material material;
+
+
+        //SliderValueText
+        public List<Text> SliderValueTextList;
+        private int x;
+
+        //parameter
+        private float somootheness;
+        private float roundcornermargin;
+        private float roundcornerradius;
+
+        private float hoverRed;
+        private float hoverGreen;
+        private float hoverBlue;
+
+        private float InnerGlowRed;
+        private float InnerGlowGreen;
+        private float InnerGlowBlue;
+        private float innergrowpower;
+
+        private float EnvironmentXRed;
+        private float EnvironmentXGreen;
+        private float EnvironmentXBlue;
+        private float EnvironmentYRed;
+        private float EnvironmentYGreen;
+        private float EnvironmentYBlue;
+        private float EnvironmentZRed;
+        private float EnvironmentZGreen;
+        private float EnvironmentZBlue;
+
+        private float rimcolorred;
+        private float rimcolorgreen;
+        private float rimcolorblue;
+
+        private float fadebegindistance;
+        private float fadecompletedistance;
+        private float fademainvalue;
+
+        private float metalic;
+
+        private float iridescenceintensityrange;
+        private float iridescencethreshold;
+        private float iridescenceangle;
+        /// <summary>
+        /// Gets the material of the object to which this class is attached.
+        /// </summary>
+        void Start()
+        {
+            material = this.gameObject.GetComponent<Renderer>().material;
+        }
+
+        /// <summary>
+        /// MRTKStandardShader parameter value is output as text.
+        /// The value of each method is set as an argument.
+        /// </summary>
+        /// <param name="textList">Textlist</param>
+        /// <param name="xRvalue">The first parameter set in the text list</param>
+        /// <param name="xGvalue">The second parameter set in the text list</param>
+        /// <param name="xBvalue">The third parameter set in the text list</param>
+        /// <param name="yRvalue">The fourth parameter set in the text list</param>
+        /// <param name="yGvalue">The fifth parameter set in the text list</param>
+        /// <param name="yBvalue">The sixth parameter set in the text list</param>
+        /// <param name="zRvalue">The seventh parameter set in the text list</param>
+        /// <param name="zGvalue">The eighth parameter set in the text list</param>
+        /// <param name="zBvalue">The ninth parameter set in the text list</param>
+        private void SetText(List<Text> textList, float xRvalue, float xGvalue, float xBvalue, float yRvalue, float yGvalue, float yBvalue, float zRvalue, float zGvalue, float zBvalue)
```

Comment:
Looking through the example scene. I'm not seeing this used anywhere.

Reply:
It's also very confusing to read and understand for new users. Some assumptions and types built in here.